tests/test_experiment_server.py

```diff
@@ -247,6 +250,30 @@ def test_working_participants_accepted(self, a, webapp):
         )
         assert models.Question.query.all()
 
+    def test_excessive_question_text_is_blocked(self, a, webapp):
+        # The normal max length is 1000
+        resp = webapp.post(
+            "/question/{}?question=q&response={}&number=1".format(
+                a.participant().id, "x" * 1001
+            )
+        )
+        assert resp.status_code == 400
+
+        resp = webapp.post(
+            "/question/{}?question=q&response={}&number=1".format(
+                a.participant().id, "x" * 1000
+            )
+        )
+        assert resp.status_code == 200
+
+        # This part of the test isn't working because of some weird use of the config object
```

Comment:
The pattern used elsewhere (which would work here, larger issues aside) is to use the `active_config` fixture:
```
        # Override the length to go shorter
        with active_config.override({"question_max_length": 99}):
           resp = webapp.post(
               "/question/{}?question=q&response={}&number=1".format(a.participant().id, 'x'*100)
           )
        assert resp.status_code == 400
```

Reply:
Good call, made that change.